x-pack/legacy/plugins/ml/public/application/routing/routes/data_frame_analytics/analytics_job_exploration.tsx

```diff
@@ -34,7 +34,7 @@ export const analyticsJobExplorationRoute: MlRoute = {
 
 const PageWrapper: FC<PageProps> = ({ location, deps }) => {
   const { context } = useResolver('', undefined, deps.config, basicResolvers(deps));
-  const { _g }: Record<string, any> = parse(location.search, { sort: false });
+  const { _g }: Record<string, any> = parse(location.search);
```

Comment:
`parse` isn't trimming the `?` char from the beginning of `location.search` and so the first parameter returned contains this prefix. `?_g` in this example.
I see some plugins are trimming this themselves or taking a substring before parsing. This will need to happen to all of ML's usage of `parse`